<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>442</width>
    <height>585</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>401</width>
     <height>521</height>
    </rect>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>311</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>311</width>
      <height>491</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,80);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLineEdit" name="username_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>250</y>
      <width>251</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,120);
font: 10pt &quot;Segoe UI&quot;;
border:none;
border-bottom:1px solid white;
color:white;
padding:12px;</string>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="password_input">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>300</y>
      <width>251</width>
      <height>40</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,120);
font: 10pt &quot;Segoe UI&quot;;
border:none;
border-bottom:1px solid white;
color:white;
padding:12px;</string>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>358</y>
      <width>161</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_4{
background:transparent;
color: rgb(255, 255, 255);
}
QPushButton#pushButton_4:hover{
	color: rgb(234, 234, 234);
}
QPushButton#pushButton_4:pressed{
padding-left:5px;
padding-top:5px;
color: white;;
}
</string>
    </property>
    <property name="text">
     <string>Forgot password? Reset Now</string>
    </property>
   </widget>
   <widget class="QPushButton" name="login_button">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>390</y>
      <width>251</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>12</pointsize>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:rgba(0,0,0,120);
	color: rgb(255, 255, 255);
	font: 12pt &quot;Segoe UI&quot;;
}</string>
    </property>
    <property name="text">
     <string>SIGN IN</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>113</x>
      <y>470</y>
      <width>181</width>
      <height>21</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton#pushButton_5{
background:transparent;
color: rgb(255, 255, 255);
}
QPushButton#pushButton_5:hover{
	color: rgb(234, 234, 234);
}
QPushButton#pushButton_5:pressed{
padding-left:5px;
padding-top:5px;
color: white;;
}
</string>
    </property>
    <property name="text">
     <string>Don't have an account? SIGN UP</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>10</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgba(0,0,0,75);</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="close_button">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>10</y>
      <width>31</width>
      <height>31</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI</family>
      <pointsize>10</pointsize>
      <italic>false</italic>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:transparent;
	color: rgb(255, 255, 255);

}</string>
    </property>
    <property name="text">
     <string>X</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>91</x>
     <y>98</y>
     <width>201</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(255,255,255,200)</string>
   </property>
   <property name="text">
    <string>22/584 Hariom Industries,
Ganesh nagar, 
2nd Lane, Ichalkaranji 416118</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>76</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Consolas</family>
     <pointsize>10</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 700 10pt &quot;Consolas&quot;;</string>
   </property>
   <property name="text">
    <string>ENTERPRISES</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_10">
   <property name="geometry">
    <rect>
     <x>86</x>
     <y>154</y>
     <width>251</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>10</pointsize>
     <bold>false</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color:rgba(255,255,255,200)</string>
   </property>
   <property name="text">
    <string> Customer Suport - +91 7769038180
 Email - mr.shubhamlohar952@gmail.com</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>92</x>
     <y>48</y>
     <width>211</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Segoe UI</family>
     <pointsize>18</pointsize>
     <italic>false</italic>
     <bold>true</bold>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 800 18pt &quot;Segoe UI&quot;;</string>
   </property>
   <property name="text">
    <string>HARIOM</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
